apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -481,7 +481,6 @@ const DisableTether = () => {
     position: 'above',
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 37 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28045&commitId=9e8da921461d2f0bfc6d8b66948cceedb8ebf0b1&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1907 screenshots unchanged, 0 screenshots with different dimensions and 35 screenshots with visible difference.


  <details><summary><b>unknown</b> 37 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - Dark Mode.active.chromium.png|3452| Changed|
|InfoButton.default - High Contrast.active.chromium.png|4045| Changed|
|InfoButton.default.active.chromium.png|2326| Changed|
|InfoButton.size.rest.chromium.png|5421| Changed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|1108| Changed|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|1380| Changed|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|1680| Changed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|10879| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|5040| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|3127| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|3129| Changed|
|Portal.Apply Class Names.should have green border.chromium.png|643| Changed|
|Positioning.auto size.chromium.png|5923| Changed|
|Positioning.explicit overflow padding - RTL.chromium.png|1113| Changed|
|Positioning.explicit overflow padding.chromium.png|1113| Changed|
|Positioning.horizontal overflow - RTL.chromium.png|682| Changed|
|Positioning.horizontal overflow padding - RTL.chromium.png|641| Changed|
|Positioning.horizontal overflow padding.chromium.png|641| Changed|
|Positioning.horizontal overflow.chromium.png|682| Changed|
|Positioning.vertical overflow padding.chromium.png|772| Changed|
|Positioning.vertical overflow.chromium.png|809| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|342| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|907| Changed|
|Tooltip Converged.basic.chromium.png|528| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|786| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|1384| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|7612| Changed|
|Tooltip Converged.inverted.chromium.png|4419| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|6145| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|2289| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|2288| Changed|
|Tooltip Converged.text-wrapping.chromium.png|1634| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|473| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|1177| Changed|
|Tooltip Converged.withArrow.chromium.png|716| Changed|
|Positioning (no decorator).scroll jumps.chromium.png|0| Added|
|Positioning (no decorator).scroll jumps.positions without scroll jump.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>